packages/salesforcedx-vscode-apex/src/commands/forceApexTestRunCodeAction.ts

```diff
@@ -33,8 +34,8 @@ function isNotEmpty(value: string): boolean {
 // cache last test class and test method values to
 // enable re-running w/o command context via built-in LRU
 class ForceApexTestRunCacheService {
```

Comment:
I might suggest moving the cache service into its own file since multiple commands are utilizing it.

Reply:
Sure. It'll be in the updated PR.

Reply:
Moved into its own file testRunCache/forceApexTestRunCacheService.ts